src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Support create/update file share with access tier
+    - `New-AzRmStorageShare`
+    - `Update-AzRmStorageShare`
```

Comment:
```suggestion
    - `New-AzStorageShare`
    - `Update-AzStorageShare`
```

Reply:
The change log is right.
We have 2 set of File share cmdlet:
1. Dataplane , like New-AzStorageShare
2. Mgmt plane, like New-AzRmStorageShare.

The new properties only support on Mgmt plane, so should change the cmdlet name like "*-RmStorageShare"